### ISSUE TITLE
Support fully localizing Scripture References

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#736]

**Comment by irahopkinson** [comment]
Maybe I'm misunderstanding but it seems like a bad idea to localize a Scripture reference. References should be absolute and unique and not dependent on the users UI language. Sure localize what the user sees but don't localize a reference.

**Comment by tjcouch-sil** [comment]
Agreed; this is just for the reader-facing locale-based reference. I'm talking about (for example; I don't know where else it's used) the reformatted reference that gets put onto the exported USX for cross references and such in the project's native reference format. I don't know what to call that, though. I will add a screenshot of the dialog in hopes that it may also clarify.

Let me know if you have other thoughts or concerns.
